.eslintrc

```diff
@@ -311,7 +311,13 @@
       ],
       "rules": {
         "max-len": "off"
+      },
+			
+      "files": ["scripts/**/*"],
+      "rules": {
+        "import/no-extraneous-dependencies": "off",
+        "no-console": "off"
```

Comment:
Can we refactor the code to remove the warning or use minimalistic console messages instead of turning off the rule?

Reply:
@akshatnema 
The scripts have `console.log` statements to **give feedbacks** on the CLI, so we need to keep the `console.log` statements in the scripts. If I consider removing the these statements I won't be able to find a any redundant console.logs. I think will have to remove most of the eslint warnings by adding a comment to ignore eslint warning before each of the log statement, 

I have added the rule to avoid adding so many comments to ignore the eslint rule. 

@akshatnema Should I remove the rule from the config and add comments (to ignore the warning) on each of log statement anyways?

![image](https://github.com/user-attachments/assets/b525a84e-3c99-4788-bfc1-da8b961d0c21)